### ISSUE TITLE
Admin notice

### ISSUE THREAD

**Issue by samikeijonen** [devinsays/edd-theme-updater#13]
For basic user adding the theme license can be a challenge. Should we add admin notice if license have not been set?

Something like "Click here to enter your license key for getting auto updates".

There could also be admin notice when license have expired.

**Comment by devinsays** [comment]
Yes, we could display a notice in the admin panel and on the update screen. I noticed Yoast plugins do this.

Do you think it should be dismissable? This adds additional complexity.

"The license key for [theme name] is required for updates. Add it here."

**Comment by samikeijonen** [comment]
I'm not sure what dismissable means. Is this ticket related?

https://core.trac.wordpress.org/ticket/31233

**Comment by devinsays** [comment]
Allow the user to hide the notice: http://wptheming.com/2011/08/admin-notices-in-wordpress/

**Comment by samikeijonen** [comment]
Got it! 

I'm reading the [ticket](https://core.trac.wordpress.org/ticket/31233) right now and I think in WP 4.2 you can dismiss admin notices.
